File: 6KYU/Calculate the area of a regular n sides polygon inside a circle of radius r/solution.go
package kata

import (
	"math"
)

const PI = 3.141592653589793 // Use this value as pi in your calculations if necessary

func AreaOfPolygonInsideCircle(circleRadius float64, numberOfSides int) float64 {
	return math.Round((((circleRadius*circleRadius)*float64(numberOfSides))/2.0)*(math.Sin((PI*2.0)/float64(numberOfSides)))*1000) / 1000
}

File: 6KYU/Calculate the area of a regular n sides polygon inside a circle of radius r/solution.c
#include <math.h>

double PI = 3.141592653589793; // Use this value as pi in your calculations if necessary
double area_of_polygon_inside_circle(double circle_radius, int number_sides) {
  return round((((circle_radius * circle_radius) * number_sides) / 2.0) * (sin((PI * 2.0)/ number_sides)) * 1000) / 1000;
}
